components/camera/videosource/webcam.go

```diff
@@ -225,8 +247,14 @@ func NewWebcam(
 	if err := cam.Reconfigure(ctx, deps, conf); err != nil {
 		return nil, err
 	}
-	cam.Monitor()
 
+	if cam.conf.FrameRate != 0.0 {
+		defaultFrameRate = cam.conf.FrameRate
```

Comment:
The `defaultFrameRate` is global. If you have 2 webcam components whose configs have different frame rates, one of them is gonna have a bad time with this. I'd make the global one `const`, and then have each camera store its own frame rate inside its struct somewhere.